.github/scripts/run-online-transducer.sh

```diff
@@ -204,6 +211,17 @@ time ./build/bin/sherpa-online \
   $repo/test_wavs/DEV_T0000000001.wav \
   $repo/test_wavs/DEV_T0000000002.wav
 
-rm -rf $repo
 log "End of testing ${repo_url}"
 log "=========================================================================="
+
+# For Endpoint
+for m in greedy_search modified_beam_search fast_beam_search; do
+  time ./build/bin/sherpa-online \
+    --decoding-method=$m \
+    --nn-model=$repo/exp/cpu_jit.pt \
+    --tokens=$repo/data/lang_char/tokens.txt \
+    --use-endpoint=true \
+    $long_audio_repo/10-minutes.wav
```

Comment:
10-minutes.wav is for English, while this model is for Chinese. I am afraid it is not able to recognize the wave.